src/nodes/CodeBlockView.vue

```diff
@@ -239,7 +239,9 @@ export default {
 	align-items: stretch;
 	position: absolute;
 	right: 12px;
-	margin-top: 4px;
+	margin: 0;
+	top: 50%;
+	transform: translateY(-50%);
```

Comment:
Not to sure about this, if the code block has multiple lines i think it should still top align. Maybe we can just reduce the height so that it is vertically centered if it is empty/one line?

Reply:
why then not to change `margin-top` to 8px instead of 4px?

Reply:
Sure; sounds good